source/_elephpants/2019-07-04-blue-check24.md

```diff
@@ -0,0 +1,20 @@
+---
+name: JePHPry
+categories:
+    - check24
+tags:
+    - blue
+sponsor: CHECK24
+reverse: CHECK24 logo
+photos:
+    -
+        file: blue-check24-0-400x300.jpg
+        link: https://twitter.com/CHECK24_JePHPry
```

Comment:
These links should be to the source of the photo, not a generic link for the elephpant (which is include below).

Reply:
I used the twitter image links but I also added the images in the photos directory. Should I leave them there or should I remove them?